From: =?utf-8?q?Sandro_Knau=C3=9F?= <hefee@debian.org>
Date: Tue, 15 Jan 2019 22:21:35 +0100
Subject: Don't modify build flags within the package

Origin: debian
Forwarded: not-needed
Last-Update: 2021-01-03

For being able to have full control over the build process, we need
full control over the build flags set. That's why it is recommended to disable
any modification from these flags from package side.
---
 src/csync/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/csync/CMakeLists.txt b/src/csync/CMakeLists.txt
index 26d66e6..aa75ab4 100644
--- a/src/csync/CMakeLists.txt
+++ b/src/csync/CMakeLists.txt
@@ -9,7 +9,6 @@ set(LIBRARY_SOVERSION "0")
 # add definitions
 include(DefineCMakeDefaults)
 include(DefinePlatformDefaults)
-include(DefineCompilerFlags)
 include(DefineOptions.cmake)
 
 include(DefineInstallationPaths)
